README.md

```diff
@@ -25,33 +25,33 @@ If for a given file there is only either a single .patch or a single .override t
 MigTool is designed to be embedded in the target app. Use the following idiom: 
 
 ```groovy
-    MigTool tool = new MigTool();
+    MigTool tool = new MigTool.Builder()
```

Comment:
We should remember that users of the library (such as Platform) will need to be updated to the new building interface once the new version is applied to them.

Reply:
I think users will notice when they upgrade! 😃 Luckily it is a very small change.